frontend/src/scenes/session-recordings/player/controller/PlayerSeekbarPreview.tsx

```diff
@@ -76,9 +74,7 @@ export function PlayerSeekbarPreview({ minMs, maxMs, seekBarRef, activeMs }: Pla
 
     const isHovering = useIsHovering(seekBarRef)
 
-    const { featureFlags } = useValues(featureFlagLogic)
-    const alwaysShowSeekbarPreview = !!featureFlags[FEATURE_FLAGS.ALWAYS_SHOW_SEEKBAR_PREVIEW]
```

Comment:
Looks like this feature flag didn't exist in PostHog so this condition would always have been false. @pauldambra may remember what it was used for

Reply:
Ah, we overlapped... 

if it doesn't exist in PostHog then we're safe to remove it. We never needed the debug functionality 👍